src/data/queries/organizationRelatedContentQuery.ts

```diff
@@ -1,9 +1,41 @@
 import { gql, useQuery } from "@apollo/client";
 import { workConnection, workFragment } from "src/data/queries/queryFragments";
-import { QueryData, QueryVar } from "./organizationQuery";
+import { Works } from "src/data/types";
+import { useSearchDoiQuery } from "./searchDoiQuery";
+import { useSearchDoiFacetsQuery } from "./searchDoiFacetsQuery";
 
 
 export function useOrganizationRelatedContentQuery(variables: QueryVar) {
+  const id = 'ror.org/' + variables.id
+  const rorId = `"https://${id}"`
+  const query = `organization_id:${id} OR affiliation_id:${id} OR related_dmp_organization_id:${id} OR provider.ror_id:${rorId}` + (variables.filterQuery ? ' AND ' + variables.filterQuery : '')
```

Comment:
I think you might need paratheses around the organization part.
Boolean logic can be weird.  I think AND has higher order of operations